pages/querying/differences-in-cypher-implementations.mdx

```diff
@@ -0,0 +1,307 @@
+---
+title: Differences in Cypher implementation
+description: Compare and contrast different Cypher implementations in Memgraph. Comprehensive documentation to level up your graph computing expertise.
```

Comment:
```suggestion
description: Compare and contrast different Cypher implementations in Memgraph. 
```

Reply:
Sounds auto-generated, not sure is it intentional :D

Reply:
TBH, didn't even look at it, it's old. I agree, will update it